Solutions/Cisco ETD/Data Connectors/azuredeploy_CiscoETD_API_FunctionApp.json

```diff
@@ -0,0 +1,324 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "FunctionName": {
+            "defaultValue": "CiscoETD",
```

Comment:
Hello @sepintocisco, Please concat function app name with resource group or subscription id. 
Reason being There can be multiple storage account with similar names. 

You can create a Variable, And put value as to concat functionapp name with other string. 

Please refer to this data connector for more clarification. 
Thanks.

Reply:
Sure, will do it.